### ISSUE TITLE
CCSH - Cannot exit via CTRL+C

### ISSUE THREAD

**Issue by ce-bo** [MaibornWolff/codecharta#3355]
# Bug

## Expected Behavior

<GIVEN>

WHEN
I run `ccsh` command in my shell and enter CTRL+C to stop the command 

THEN
I want the command to be exited

## Actual Behavior
Nothing happens, I am stuck.

## Steps to Reproduce the Problem
1.
1.
1.

## Specifications
- is released in [online-demo](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json.gz&file=codecharta_analysis.cc.json.gz): yes | no
- CodeCharta Version: 1.119.0
- OS: Windows (cmd.exe, powershell, but not git bash).
- Browser: -

**Comment by moritz-suckow** [comment]
I took a look at it and this seems to be correlated to KotlinInquirer. KInquirer uses Jline3 to handle console input in a rather old version (3.21.0). There was a very similar issue to this one at Jline3 (https://github.com/jline/jline3/issues/822) which was closed and might have fixed the problem. Since then two new versions of Jline3 were released, but KotlinInquirer did not update their dependency. 
I am unsure if we can fix this from our side.

**Comment by fritschldwg** [comment]
Following up on the previous comment I forked the kotlin-inquirer repo with the following takeaways:

- Just updating jline to 3.23+ doesn't fix the issue on Windows. Control-C still propagates to the kotlin-inquirer and causes errors, but can be fixed with little code changes
- Since it's unlikely that any PRs will be merged into the kotlin-inquirer repo (some from 2022 are still open), we could use fork in codecharta. This would also let us fix backspacing in gitbash, which currently doesn't work.

**Comment by fritschldwg** [comment]
We could use one of the following solutiosn to address the shortcomings of the kotlin-inquirer:
- Fork the kotlin-inquirer repository and make necessary modifications
- Develop our own inquirer using kotter library (https://github.com/varabyte/kotter)
- Incorporate a java-inquirer into our project

**Comment by ce-bo** [comment]
I think this is blocked by #3483

**Comment by BenediktMehl** [comment]
Fixed with #3609
